libnymea-zigbee/zdo/zigbeedeviceprofile.h

```diff
@@ -198,12 +198,23 @@ class ZigbeeDeviceProfile
     Q_ENUM(DeviceType)
 
     enum Relationship {
-        Parent,
-        Child,
-        Sibling
+        RlationShipParent,
```

Comment:
`RlationShip*` -> `Relationship*` ?

Since the ENUM is called `Relationship` I would also suggest to not make `RelationShip` in the values

Reply:
done